e/index.html

```diff
@@ -331,11 +331,11 @@ <h1><span>About OpenToonz</span></h1>
 			<div class="inner clearfix">
 				<div class="card block left">
 					<ul class="list">
-						<li>This is software for producing a 2D animation.
-							<p class="notes"> It is based on the software "Toonz", which was developed by Digital Video S.p.A. in Italy, customized by Studio Ghibli, and has been used for creating its works for many years. Dwango launches this OpenToonz project, in cooperation with Digital Video and Studio Ghibli.</p></li>
-						<li>This software can be used by anyone free of charge, no matter whether the purpose of use is commercial or not.</li>
-						<li>This is available as an open source, so that users can modify its source code freely.</li>
-						<li>We aim to develop a new platform for connecting the academic research into frontline animation production.</li>
+						<li>Software for the production of 2D animation.
+							<p class="notes"> Based on the software "Toonz", developed by Digital Video S.p.A. in Italy, OpenToonz has been customized by Studio Ghibli, and used for the creation of its works for many years. Dwango has launched the OpenToonz project in cooperation with Digital Video and Studio Ghibli.</p></li>
+						<li>OpenToonz can be used free of charge for both commercial and non-commercial projects.</li>
+						<li>Available as an open-source so that anyone can modify its source code freely.</li>

```

Comment:
Maybe "Available under an open-source license so that anyone can modify its source code freely."?